jobs/delorean/misc/github-events/Jenkinsfile

```diff
@@ -1,46 +1,101 @@
-        when(isGABranch || isRCBranch || isLatestBranch) {
+    stage('Trigger branch job') {
+        when(runBranchJob) {
+            println "Run branch job for ${productName}/${branchType}"
             def branchType = gitBranch.split("-").last()
             def productName = gitBranch.minus("-${branchType}")
-            jobName = "delorean-${productName}/${branchType}/branch"
+            def jobName = "delorean-${productName}/${branchType}/branch"
             build job: jobName, wait: false
         }
     }
 
+    stage('Trigger integration test') {
+        when(runIntegrationTest) {
+            println "Run integration test with ${installationGitUrl} ${gitBranch}"
+            withCredentials([usernamePassword(
+                    credentialsId: githubUserPassCredentialsID,
+                    passwordVariable: 'GITHUB_PASSWORD',
+                    usernameVariable: 'GITHUB_USERNAME')]) {
+                final GitHub gitHub = new GitHubBuilder()
+                        .withOAuthToken(env.GITHUB_PASSWORD, env.GITHUB_USERNAME)
+                        .build()
+                ghUtils.ghUpdatePrCommitStatus(gitHub, prUrl, GHCommitState.PENDING, env.BUILD_URL, 'Pending', ghCommitStatusContext)
+            }
+            try {
+                def jobName = "openshift-cluster-integreatly-test"
+                def jobParams = [
+                    [$class: 'StringParameterValue', name: 'installationGitUrl', value: installationGitUrl],
+                    [$class: 'StringParameterValue', name: 'installationGitBranch', value: gitBranch]
+                ]
+                build job: jobName, parameters: jobParams
+            } catch (Exception e) {
+                withCredentials([usernamePassword(
+                        credentialsId: githubUserPassCredentialsID,
+                        passwordVariable: 'GITHUB_PASSWORD',
+                        usernameVariable: 'GITHUB_USERNAME')]) {
+                    final GitHub gitHub = new GitHubBuilder()
+                            .withOAuthToken(env.GITHUB_PASSWORD, env.GITHUB_USERNAME)
+                            .build()
+                    ghUtils.ghUpdatePrCommitStatus(gitHub, prUrl, GHCommitState.FAILURE, env.BUILD_URL, 'Failure', ghCommitStatusContext)
+                    error "Integration test failures, ${installationGitUrl}, ${gitBranch}, ${e}"
+                }
+            }
+            withCredentials([usernamePassword(
+                    credentialsId: githubUserPassCredentialsID,
+                    passwordVariable: 'GITHUB_PASSWORD',
+                    usernameVariable: 'GITHUB_USERNAME')]) {
+                final GitHub gitHub = new GitHubBuilder()
+                        .withOAuthToken(env.GITHUB_PASSWORD, env.GITHUB_USERNAME)
+                        .build()
+                ghUtils.ghUpdatePrCommitStatus(gitHub, prUrl, GHCommitState.SUCCESS, env.BUILD_URL, 'Success', ghCommitStatusContext)
+            }
+        }
+    }
```

Comment:
You shouldn't need to add this block, triggering the branch job will do this since it checks the PR labels https://github.com/integr8ly/ci-cd/blob/master/jobs/delorean/jenkinsfiles/branch/ga/Jenkinsfile#L97